src/index.html

```diff
@@ -1,12 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wbpack Exercise</title>
-  </head>
-  <body>
-    <h1>Hello webpack!</h1>
-    <h2>this is test again right now?</h2>
-  </body>
-</html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="dflex">
+        <img id="logo" alt="Chicken Food">
+        <ul class="dflex">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </header>
+    <main class="dflex">
+        <section>
+            <h1 id="mainTitle"></h1>
```

Comment:
- [ ] Is this an empty tag? If you do not have any content to add to it or injecting any DOM actions into it, I suggest you remove it but for the page's accessibility and good SEO, I will suggest you include a major header that summarizes what the page is about - Maybe the name of the page or something :thinking: 
- That is best practices...for us! :hugs: